modules/experimental/pages/ff-blast.adoc

```diff
@@ -0,0 +1,15 @@
+= FF Blast
+
+`Detachment ability`
+
+Some forces are able to disrupt their firefight opponents, for example by using the sonic weapons of Slaaneshi Noise Marines.
+
+---
+
+Where your detachment has the {ff-blast} ability and takes part in a firefight:
+
+* After rolling to resolve the firefight, every enemy detachment that was within 15 cm of your detachment gains one Blast marker. This effect does not stack with other detachments with the {ff-blast} ability.
```

Comment:
I thought I had an edge case to consider here, but it looks like line of fire doesn't come into the definition of 'engaged', so that keeps things simpler, phew! :-D

So, just a little neatening:

```suggestion
* After you roll for assault resolution in this firefight, place one additional Blast marker on each enemy detachment that was within 15 cm of this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```

Reply:
As with CC Blast, 'additional' isn't appropriate.

If we refer directly to the step again:

```suggestion
* At the end of xref:roll-to-resolve-the-firefight.adoc[], place one Blast marker on each enemy detachment that was within 15 cm of this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```

Reply:
Yup, I take your point again. Similar to our discussion on CC Blast:

```suggestion
* At the end of the xref:main-rules:roll-to-resolve-the-firefight.adoc[] step, place one Blast marker on each enemy detachment that had any units within 15 cm of units in this detachment.
This effect does *not* stack with other detachments that also have the {ff-blast} ability.
```